time timestamp
int8  node_id
int8 anchor_id
float32 distance
int16 confidence
